auth/auth_stub.go

```diff
@@ -22,7 +22,12 @@ package auth
 
 import "context"
 
-var _ Client = &noop{}
+var (
+	// NoopClient is used for testing and no-op integration
+	NoopClient = noopClient(nil)
```

Comment:
yarpc does `Nop`, we do `Noop`, and we also have `NullHost`....

How do we consolidate?